cms/migrations/0022_auto_20180719_1638.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2018-07-19 16:38
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contenttypes', '0002_remove_content_type_name'),
+        ('cms', '0021_auto_20180507_1432'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='placeholder',
+            name='content_type',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, to='contenttypes.ContentType'),
```

Comment:
This line is too long. Paulo will pick up on this.

Reply:
PEP8 recommends a line length of 79 which most organizations consider as too short and usually use 120 instead. In the Django CMS codebase as well I find that a lot of lines violate the constraint of 80 characters.  In FIL and Divio is there a defacto line length limit?

Reply:
I wouldn't touch anything that generated as part of migration script.